docker/site.sh

```diff
@@ -5,7 +5,7 @@ cd /consistency/cms_consistency/site_cmp3
 cp /opt/proxy/x509up /tmp/x509up
 chmod 600 /tmp/x509up
 export X509_USER_PROXY=/tmp/x509up
-export RUCIO_CONFIG=/consistency/rucio-client.cfg
+export RUCIO_CONFIG=/opt/rucio/etc/rucio.cfg
```

Comment:
`/opt/rucio/etc/rucio.cfg` already exist in the container and is created by `docker/rucio.cfg.j2`. This file can be used instead of `/consistency/rucio-client.cfg` which is hard coded. This way environment variables can be used to define the appropriate configuration, that will be a bit different for integration and production.

It is also noted that the information stored in `/consistency/rucio-client.cfg` was also moved to `docker/rucio.cfg.j2`.

Reply:
Is the current state of `rucio-client.cfg` the same as the derived `rucio.cfg`?